<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemSettingsMapEditor</class>
 <widget class="QDialog" name="ItemSettingsMapEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование настроек отображения</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="resetToDefBtn">
       <property name="toolTip">
        <string>Сброс по-умолчанию</string>
       </property>
       <property name="statusTip">
        <string>Сброс по-умолчанию</string>
       </property>
       <property name="whatsThis">
        <string>Сброс по-умолчанию</string>
       </property>
       <property name="text">
        <string>Сброс по-умолчанию</string>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="loadButton">
       <property name="toolTip">
        <string>Загрузить настройки из файла</string>
       </property>
       <property name="statusTip">
        <string>Загрузить настройки из файла</string>
       </property>
       <property name="whatsThis">
        <string>Загрузить настройки из файла</string>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="saveButton">
       <property name="toolTip">
        <string>Сохранить настройки в файл</string>
       </property>
       <property name="statusTip">
        <string>Сохранить настройки в файл</string>
       </property>
       <property name="whatsThis">
        <string>Сохранить настройки в файл</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="applyButton">
       <property name="toolTip">
        <string>Применить и закрыть</string>
       </property>
       <property name="statusTip">
        <string>Применить и закрыть</string>
       </property>
       <property name="whatsThis">
        <string>Применить и закрыть</string>
       </property>
       <property name="text">
        <string>Применить и закрыть</string>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="cancelButton">
       <property name="toolTip">
        <string>Закрыть</string>
       </property>
       <property name="statusTip">
        <string>Закрыть</string>
       </property>
       <property name="whatsThis">
        <string>Закрыть</string>
       </property>
       <property name="text">
        <string>Закрыть</string>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionLoadLayout1">
   <property name="text">
    <string>Загрузить Набор 1</string>
   </property>
   <property name="toolTip">
    <string>Загрузить Набор 1</string>
   </property>
  </action>
  <action name="actionLoadLayout2">
   <property name="text">
    <string>Загрузить Набор 2</string>
   </property>
   <property name="toolTip">
    <string>Загрузить Набор 2</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
